ocaml/xapi/pool_features_helpers.ml

```diff
@@ -0,0 +1,108 @@
+(*
+ * Copyright (C) 2024 Cloud Software Group
```

Comment:
lower case (c)

Reply:
The “C” in the copyright symbol © is typically uppercase when used in the symbol itself